src/cascadia/TerminalApp/TerminalSettingsSerializationHelpers.h

```diff
@@ -231,6 +235,11 @@ struct ::TerminalApp::JsonUtils::ConversionTrait<::TerminalApp::LaunchPosition>
     {
         return json.isString();
     }
+
+    std::string TypeName() const

```

Comment:
Huh, `TypeName` doesn't really seem to make sense as the method name for the situation of something like this, but I suppose it's good enough.

`TypeLabel`?

Reply:
TypeDescription perhaps